Cargo.toml

```diff
@@ -31,3 +31,4 @@ tokio-core = { version = "~0.1", optional = true }
 [dev-dependencies]
 version-sync = "~0.5"
 etherparse = "0.9.0"
+serial_test = "0.3.2"
```

Comment:
Hey, this is cool :-). Didn't know about that one.

I've noticed version 0.4 came out yesterday, maybe this could be used.

Anyway, looking at the dependencies, I know this is only a style point (valid for the etherparse above too):

* I prefer to prefix them with the tilde (`~`). This is the same as no prefix, but it is cleaner in stating the version required is not *exact*, but only semver-compatible.
* I omit the last digit that's not needed, to give the resolver as much freedom as possible. For etherparse, the `0` at the end is not needed for sure. Do you need some specific changes/bugfixes in `serial_test 0.3.2`, or any `0.3` would do? (or 0.4)?